gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -355,6 +361,11 @@ public GoogleCloudStorageImpl(
 
     this.storageRequestAuthorizer = initializeStorageRequestAuthorizer(storageOptions);
     this.downscopedAccessTokenFn = downscopedAccessTokenFn;
+
+    // Set GSC connector logging-level based on the passed debug logging option.
```

Comment:
nit: Remove the comment (that serves no purpose really)

Reply:
Removed